ui/packages/catalog-ui-search/src/main/webapp/react-component/inputs/enum-input/enum-input.js

```diff
@@ -53,23 +67,49 @@ const EnumInput = ({
           onChange={setInput}
         />
       </TextWrapper>
-      <Menu value={value} onChange={onChange}>
+      <Menu value={value} onChange={onChange} class="fa">
         {allowCustom &&
           displayInput && (
             <EnumMenuItem value={input}>{input} (custom)</EnumMenuItem>
           )}
         {filteredSuggestions.map(suggestion => {
           return (
-            <EnumMenuItem key={suggestion.value} value={suggestion.value}>
+            <EnumMenuItem
+              title={
+                isAttributeDisabled(settingsModel,suggestion.value)
```

Comment:
```suggestion
                isAttributeDisabled(settingsModel, suggestion.value)
```
brought to you by `/projects/ddf/ui>> yarn format -w`